docs/resources/servicestage_component_instance.md

```diff
@@ -0,0 +1,420 @@
+  The [object](#servicestage_command_param) structure is documented below.
+
+* `http_param` - (Optional, List) Specifies the commands. Required if `type` is **http**.
+  The [object](#servicestage_http_param) structure is documented below.
+
+* `tcp_param` - (Optional, List) Specifies the commands. Required if `type` is **tcp**.
+  The [object](#servicestage_tcp_param) structure is documented below.
+
+* `delay` - (Optional, Int) Specifies the interval between the startup and detection.
+
+* `timeout` - (Optional, Int) Specifies the detection timeout interval.
+
+<a name="servicestage_command_param"></a>
+The `command_param` block supports:
+
+* `commands` - (Required, List) Specifies the command list.
+
+<a name="servicestage_http_param"></a>
+The `http_param` block supports:
+
+* `protocol` - (Required, String) Specifies the protocol. The valid values are `HTTP` and `HTTPS`.
+
+* `port` - (Required, String) Specifies the port number.
+
+* `path` - (Required, String) Specifies the request path.
+
+* `host` - (Optional, String) Specifies the custom IP address. The defualt address is pod IP address.
+
+<a name="servicestage_tcp_param"></a>
+The `tcp_param` block supports:
+
+* `port` - (Optional, String) Specifies the port number.
+
+<a name="servicestage_external_access"></a>
+The `external_access` block supports:
+
+* `protocol` - (Optional, String) Specifies the protocol. The valid values are **HTTP** and **HTTPS**.
+
+* `address` - (Optional, String) Specifies the access address. For example: **www.example.com**.
+
+* `port` - (Optional, String) Specifies the listening port of the application component process.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The application ID in UUID format.
+
+* `status` - The instance status, which supports:
+  + **FAILED**
+  + **RUNNING**
+  + **DOWN**
+  + **STOPPED**
+  + **UNKNOWN**
+  + **PARTIALLY_FAILED**
+
+## Import
+
+Instances can be imported using their related `application_id`, `component_id` and `id`, separated by a slash (/), e.g.
+
+```
+terraform import huaweicloud_servicestage_component_instance.test 4e65a759-e7b1-4e9e-8277-857f8e261f3c/4e65a759-e7b1-4e9e-8277-857f8e261f3c/c0a13d88-d4e3-11ec-93a9-0255ac101d30
+```
+
+Note that the imported state may not be identical to your resource definition, due to some attrubutes missing from the
+API response, security or some other reason. The missing attributes is `configuration[0].env_variable`.
+It is generally recommended running `terraform plan` after importing an instance.
+You can then decide if changes should be applied to the instance, or the resource definition should be updated to
+align with the instance. Also you can ignore changes as below.
+
+```
+resource "huaweicloud_servicestage_component_instance" "test" {
+    ...
+
+  lifecycle {
+    ignore_changes = [
+      configuration[0].env_variable,
+    ]
+  }
+}
+```
+
+## Timeouts
```

Comment:
we usually put `Timeouts` above `Import` block

Reply:
done.